### DISCUSSION TITLE
Self hosting with starlink, relays and NASes

### DISCUSSION THREAD

**Discussion by markofthewild** [AzuraCast/AzuraCast#7457] [Q&A]
posted in part on the general category. if this is not allow,mods let me know please.

starlink wont let me port forward but i do have my own site. is there a way to just send the stream through my site. remote relays perhaps? i have tried install azurarelay (in my linux box that has docker and azuracast working locally) but i keep getting this error

curl -L https://raw.githubusercontent.com/AzuraCast/AzuraRelay/main/docker.sh > docker.sh bash: docker.sh: Permission denied

is any of this possible with starlink even?

Also on the website it shows a way to install it on a synology NAS. can it be installed on other NAS's like qnap or asustor etc..

edit: i want to keep all my media locally as I have 4 tbytes of audio and dont want to upload any
I also just found out that I can port forward with my vpn but only if the exit server is not in the US so this may be a workaround

**Comment by bombelman** [comment]
did you try the mountpoint url?
example:
http(s)://domain.com/listen/mountpoint/live

**Reply by markofthewild** [reply]
no i havent set up a site. thats easy to do though. either wordpress or hard code it. but i would still need to forward a port wouldnt I? 
all i want to do is to eventually broadcast from my local host (laptop or minipc) from a NAS with 4Ts of mp3s and be able to send the stream somehow to a page on my website.
all though starlink if possible
